src/sentry/static/sentry/app/views/organizationGroupDetails/groupEventDetails.jsx

```diff
@@ -215,17 +218,63 @@ class GroupEventDetails extends React.Component {
   }
 }
 
-export {GroupEventDetails};
+class GroupEventDetailsEnvWrapper extends React.Component {
```

Comment:
can we call this `GroupEventDetailsContainer`

Reply:
How much of a pain would it be to have just this component in TS?

Reply:
I can try and convert it real quick hopefully shouldn't be too bad